config/dependencies.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * prooph (http://getprooph.org/)
+ *
+ * @see       https://github.com/proophsoftware/prooph-package for the canonical source repository
+ * @copyright Copyright (c) 2016 prooph software GmbH (http://prooph-software.com/)
+ * @license   https://github.com/proophsoftware/prooph-package/blob/master/LICENSE.md New BSD License
+ */
+
+use Zend\ServiceManager\Factory\InvokableFactory;
```

Comment:
Is this the only reason for the zend/service-manager dependency? If so we should provide our own InvokableFactory.